lib/Autograd.chpl

```diff
@@ -987,6 +985,23 @@ record negOp : serializable {
 }
 
 
+record nllLossOp : serializable {
+    var input: shared BaseTensorResource(?);
+    var target: shared BaseTensorResource(?);
+    var weight: shared BaseTensorResource(?);
+    var ignoreIndex: int;
+    var red: bool;
+    var reduction: string;
+    
+    proc children do return (input,target,weight,);
```

Comment:
Only single element tuples have an ending `,)`. This should just be `(input,target,weight)`.